lib/elixir/lib/agent.ex

```diff
@@ -140,7 +140,7 @@ defmodule Agent do
   @doc """
   Returns a specification to start an agent under a supervisor.
 
-  See `Supervisor`.
+  See `Supervisor.Spec` for more information.
```

Comment:
`Supervisor.Spec` is deprecated. We shouldn't be linking to it.

Reply:
I always get confused by these. I will fix it